projects/rero/ng-core/src/lib/record/search/record-search-page.component.html

```diff
@@ -15,6 +15,6 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <ng-core-record-search [adminMode]="adminMode" [currentType]="currentType" [types]="types" [detailUrl]="detailUrl"
-  [showSearchInput]="showSearchInput" [q]="q" [page]="page" [size]="size"
-  [sort]="sort" (parametersChanged)="updateUrl($event)">
+  [showSearchInput]="showSearchInput" [q]="q" [page]="page" [size]="size" [sort]="sort"
+  (parametersChanged)="updateUrl($event)" (esResult)="recordsResult($event)">
```

Comment:
esResult should not be configured in ng-core, only in the extended component.

Reply:
done